### PR TITLE
Bug/#459 faf icon not showing

### PR DESCRIPTION
**Pull request by Geosearchef**
Not quite happy with this fix, but as fonts are loaded automatically, I don't know how to do it differently. This way shouldn't be expensive and worked reliably for me.

### CODE REVIEW

**Comment by axel1200** [comment]
What about having font defined by attribute in Fxml file font-famliy (should be that attribute)?

File: .idea/misc.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <item index="1" class="java.lang.String" itemvalue="org.junit.Test" />
     </list>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_3" default="false" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (micheljung) [inline .idea/misc.xml]: Language level 1.3? NO? :D

File: .idea/modules.xml [hunk 2]
```diff
@@ -4,6 +4,8 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/modules/downlords-faf-client.iml" filepath="$PROJECT_DIR$/.idea/modules/downlords-faf-client.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/modules/downlords-faf-client_main.iml" filepath="$PROJECT_DIR$/.idea/modules/downlords-faf-client_main.iml" group="downlords-faf-client" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/downlords-faf-client_main.iml" filepath="$PROJECT_DIR$/.idea/modules/downlords-faf-client_main.iml" group="downlords-faf-client" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/downlords-faf-client_test.iml" filepath="$PROJECT_DIR$/.idea/modules/downlords-faf-client_test.iml" group="downlords-faf-client" />
```
Comment (micheljung) [inline .idea/modules.xml]: Please revert

File: src/main/java/com/faforever/client/login/LoginController.java [hunk 3]
```diff
@@ -52,6 +54,11 @@ public void initialize() {
     loginProgressPane.setVisible(false);
     loginErrorLabel.managedProperty().bind(loginErrorLabel.visibleProperty());
     loginErrorLabel.setVisible(false);
+
+    // fixes #459
+    // "re" loads the font for the FAF logo and warning icons on the login page
+    // these are defined in style.css ".icon" but not used if WindowController.maximize() is called at stage initialization
+    Font.loadFont(LoginController.class.getResource("/font/dfc-icons.ttf").toExternalForm(), 10);
```
Comment (micheljung) [inline src/main/java/com/faforever/client/login/LoginController.java]: I just realize that this font isn't loaded anywhere anymore, not even in the CSS? This was never intended :) You can load this font in the class `Main` like so:
```
    Font.loadFont(getClass().getResourceAsStream("/font/dfc-icons.ttf"), defaultFontSize);
```

And the Open Sans font should be loaded in the CSS file (I could swear it was there....). However, this, unfortunately, causes issues with SceneBuilder (doesn't render style anymore). I think it was something like this:

```
@font-face {
   src: url("@../font/dfc-icons.ttf");
}
```
or:
```
@font-face {
   src: url("/font/dfc-icons.ttf");
}
```

IIRC one worked in SceneBuilder, while the other worked in the program.

File: .idea/misc.xml (hunk 1 above)
Comment (Geosearchef) [inline .idea/misc.xml]: No, definetly not.

**Pull request merged**